src/common/meta/src/ddl/create_table.rs

```diff
@@ -285,13 +319,19 @@ pub struct TableCreator {
 }
 
 impl TableCreator {
-    pub fn new(cluster_id: u64, task: CreateTableTask, region_routes: Vec<RegionRoute>) -> Self {
+    pub fn new(
+        cluster_id: u64,
+        task: CreateTableTask,
+        region_routes: Vec<RegionRoute>,
+        region_topics: Vec<KafkaTopic>,
```

Comment:
Consider the ability of backward compatibility. I prefer to use an enum, such as `RegionWalConfig`.

Reply:
`region_topics` is not wrapped into a `RegionWalOptions` struct.